.gitattributes

```diff
@@ -1,2 +1,2 @@
 * linguist-vendored
-*.js linguist-vendored=false
\ No newline at end of file
+*.js linguist-vendored=true
```

Comment:
is it works??

Reply:
I think it would work.

when all *.js files were `false`, the repo only shows *.js to languages.